docker-compose.yml

```diff
@@ -226,6 +226,24 @@ services:
     command: npm run start:dev
     # if enabling debug, be sure to uncomment / expose the debug port above.
     # command: npm run start:dev:debug
+  web-comment:
```

Comment:
is it worth depending on appeals-service-api?

Reply:
or perhaps can leave until we actually need to connect to api

Reply:
Just resolving for now to get the initial merge done then we can pick these points back up